Widgets/widget-IndicatorManualUsers.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "indicators-manual-user",
+  "version": -1,
+  "name": "Manual Reputation Indicators by User",
+  "dataType": "indicators",
+  "widgetType": "bar",
+  "size": 1000,
```

Comment:
Can we have more than 1000 users? Maybe we should put a reasonable default in the server instead